src/main/java/com/sourcegraph/webhook/registry/Webhook.java

```diff
@@ -17,13 +18,47 @@
     public String endpoint;
     @Expose(serialize = false)
     public String secret;
+    @Expose
+    public String lastError;
+    @Expose
+    public Date lastEvent;
 
-    public Webhook(int id, String name, String scope, Set<String> events, String endpoint, String secret) {
+    public Webhook(int id, String name, String scope, Set<String> events, String endpoint, String secret, String lastError, Date lastEvent) {
         this.id = id;
         this.name = name;
         this.scope = scope;
         this.events = events;
         this.endpoint = endpoint;
         this.secret = secret;
+        this.lastError = lastError;
+        this.lastEvent = lastEvent;
+    }
+
+    public int getId() {
+        return this.id;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public String getEndpoint() {
+        return this.endpoint;
+    }
+
+    public String getScope() {
+        return this.scope;
+    }
+
+    public Set<String> getEvents() {
+        return this.events;
+    }
+
+    public String getLastError() {
+        return this.lastError;
+    }
+
+    public Date getLastEvent() {
+        return this.lastEvent;
```

Comment:
OMG Java boilerplate. Is there no magic meta annotation the generates these for us?